lib/franklin/articles/projector.ex

```diff
@@ -22,9 +22,28 @@ defmodule Franklin.Articles.Projector do
     "article:#{article_id}"
   end
 
-  project(%ArticleCreated{id: id}, _, fn multi ->
-    Ecto.Multi.insert(multi, :article, %Article{id: id})
```

Comment:
I can not insert an empty row in the database, so I need to do a more full `insert_changeset` with all the values.